src/privileged/FirefoxMonitor.jsm

```diff
@@ -53,6 +53,11 @@ this.FirefoxMonitor = {
   kFirefoxMonitorURLPref: "extensions.fxmonitor.FirefoxMonitorURL",
   kDefaultFirefoxMonitorURL: "https://monitor.firefox.com",
 
+  kDebugPref: "extensions.fxmonitor.debug",
+  get debug() {
+    return Preferences.get(this.kDebugPref, false),
```

Comment:
Needs to be `;` at the end, or it results in `expected expression, got '}' FirefoxMonitor.jsm:59:2`